### PR TITLE
Use stderr for console log output

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

**Comment by urso** [comment]
closes #219

File: logp/logp.go [hunk 1]
```diff
@@ -107,7 +109,7 @@ func Init(name string, config *Logging) error {
 
 func SetStderr() {
```
Comment (andrewkroh) [inline logp/logp.go]: It looks like this function is not used anywhere? Should we just remove it.

**Comment by andrewkroh** [comment]
LGTM

File: logp/logp.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ type Logging struct {
 func init() {
 	// Adds logging specific flags: -v, -e and -d.
 	verbose = flag.Bool("v", false, "Log at INFO level")
-	toStderr = flag.Bool("e", false, "Output to stdout and disable syslog/file output")
+	toStderr = flag.Bool("e", false, "Output to stderr and disable syslog/file output")
```
Comment (mrkschan) [inline logp/logp.go]: Shall this be "Log to stderr and disable syslog/file output"? Aligning to -v option.

Comment (urso) [inline logp/logp.go]: good catch

**Pull request merged**